modules/user_accounts/test/user_accountsTest.php

```diff
@@ -36,34 +62,313 @@ function testUserAccountsDoespageLoad()
+        $bodyText = $this->webDriver->findElement(
+            WebDriverBy::cssSelector("body")
+        )->getText();
         $this->assertContains("Edit User", $bodyText);
-
         $this->assertEquals(
             "password",
-            $this->webDriver->findElement(WebDriverBy::Name("Password_md5"))->getAttribute("type")
+            $this->webDriver->findElement(
+                WebDriverBy::Name("Password_md5")
+            )->getAttribute("type")
         );
         $this->assertEquals(
             "checkbox",
-            $this->webDriver->findElement(WebDriverBy::Name("NA_Password"))->getAttribute("type")
+            $this->webDriver->findElement(
+                WebDriverBy::Name("NA_Password")
+            )->getAttribute("type")
         );
         $this->assertEquals(
             "password",
-            $this->webDriver->findElement(WebDriverBy::Name("__Confirm"))->getAttribute("type")
+            $this->webDriver->findElement(
+                WebDriverBy::Name("__Confirm")
+            )->getAttribute("type")
         );
     }
-
     /**
-     * Tests that, when loading the User accounts module > my_preference submodule, some
-     * text appears in the body.
+     * Tests that, when loading the User accounts module > my_preference submodule
+     * some text appears in the body.
      *
      * @return void
      */
     function testUserAccountsMyPreferencesDoespageLoad()
     {
         $this->safeGet($this->url . "/user_accounts/my_preferences/");
-        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $bodyText = $this->webDriver->findElement(
+            WebDriverBy::cssSelector("body")
+        )->getText();
         $this->assertContains("My Preferences", $bodyText);
     }
+    /**
+     * Tests that searching for users using thei user IDs works
+     *
+     * @return void
+     */
+    function testSearchForUsers()
+    {
+        $this->safeGet($this->url . "/user_accounts/");
+        $bodyText = $this->webDriver->findElement(
+            WebDriverBy::cssSelector("body")
+        );
+        $bodyText->getText();
+        $this->_assertSearchBy(
+            array('userID' => 'my_nonexistent_user_ID'),
+            null
+        );
+        $this->_assertSearchBy(
+            array('userID' => 'UnitTester'),
+            array(self::$_UNIT_TESTER)
+        );
+        $this->_assertSearchBy(
+            array('userID' => 'unittester'),
+            array(self::$_UNIT_TESTER)
+        );
+        $this->_assertSearchBy(
+            array('userID' => 'n'),
+            array(
+             self::$_ADMIN,
+             self::$_UNIT_TESTER,
+            )
+        );
+    }
+    /**
+     * Tests various user account edit operations.
+     *
+     * @return void
+     */
+    function testUserAccountEdits()
+    {
+        $this->_verifyUserModification(
+            'user_accounts',
```

Comment:
Actually no. I access the user accounts page to check that the modifications done previously we successfully done.